src/main/java/com/andersen/users/User.java

```diff
@@ -0,0 +1,9 @@
+package com.andersen.users;
+
+import com.andersen.Ticket;
+
+public abstract class User {
+
+	protected Ticket ticket;
```

Comment:
how you will add this ticket for your object?

Reply:
sure, it's needed

Reply:
I see I need to add constructor here